src/glmfit.jl

```diff
@@ -115,7 +115,23 @@ function confint(obj::AbstractGLM, level::Real)
 end
 confint(obj::AbstractGLM) = confint(obj, 0.95)
 
-deviance(m::AbstractGLM)  = deviance(m.rr)
+deviance(m::AbstractGLM) = deviance(m.rr)
+
+function loglikelihood(m::AbstractGLM)
```

Comment:
Yeah, but even looking closely at the formulas found in various sources, I still can't figure it out. Not a big issue in practice, but still I'm curious... I'll try harder. Something that bugs me in particular is how to take into account the dispersion parameter, which isn't the same for the fitted model and for the saturated model. And yet the formulas include both under the same `σ²` term...